### DISCUSSION TITLE
Mission control error impossible to solve

### DISCUSSION THREAD

**Discussion by gabrielolguinorellana** [ndeadly/MissionControl#822] [General]
System version: 17.0.0|AMS: 1.6.2|E (Switch Oled with Picofly)

Dear community, 

I'm experiencing a very annoying problem, which started when I deleted the 010000000000bd00 folder of my 0.9.4 Mission Control by mistake.

When I realized it was crucial to connect my PS5 controller to the Switch, I downloaded and installed the files of the newest 0.11.0 version, replacing the older ones when needed. However, I got this error

```
A fatal error occurred when running Atmosphère.
Program ID: 01000000000000 Error 
Desc: std::abort() called (0xffe)
Report saved to /atmosphere/fatal_errors/report_000000000c481e6e.bin Press POWER to reboot.

```
Here, I read that it is a common error when a newer version of Mission Control is not supported by the system. So, I decided to downgrade it to 0.9.4 as it was at the begining, but the error persisted. It was the same for all tried versions, which were all the releases between 0.8 and 0.10.0. I also tried cleaning the paired devices database, but nothing improved. So I don't know what to do to solve it. On the other hand, If I remove the files that I know are related to mission control, the switch works fine except because it doesn't recognize my ps5 controllers. 

Have you experienced something similar? Is it possible that other files in the system should be related to Mission control in addition to those mentioned in the installation tutorial? Other that should be updated or replaced? https://github.com/ndeadly/MissionControl
Thank you in advance for your support! 

[report_000000000c481e6e.zip](https://github.com/ndeadly/MissionControl/files/15477961/report_000000000c481e6e.zip)

**Comment by ndeadly** [comment]
You need Atmosphere 1.7.0 to run 0.11.0. There were kernel changes in HOS 18.0.0 that Atmosphere has implemented that break compatibilty with earlier Atmosphere versions. Since Mission Control releases always target the latest atmosphere release (at the time) it inherits these limitations. Maintaining multiple concurrent releases to support outdated Atmosphere versions is not really in my interest.

**Comment by gabrielolguinorellana** [comment]
Dear ndeadly,

Thank you for your answer and congratulations for all your efforts. Your advances with Mission Control are very inspiring to me. 

Actuallly, I would be happy if 0.9.4 works again for me, but I don't know how to do it. I'm getting the error that I mentioned with all version of Mission Control and I don't understand why. In the past, I used the same version of Atmosphere and HOS together MC 0.9.2, 0.9.3 and 0.9.4 without any trouble. Nothing is different in my system now, but it crash with these version are installed. Could you help me to know the reason? 

Thank you so much for your support!

**Reply by ndeadly** [reply]
You need version 0.10.0+ to run on HOS 17.0.0. You can't have been running a 0.9.x version successfully on this firmware in the past - it would abort with the same error you have reported above.

**Comment by gabrielolguinorellana** [comment]
You were right. My error.
I solved it with 0.10.0

Thank you so much!!
